SPECS/gcc/gcc.spec

```diff
@@ -2,15 +2,15 @@
 %define _use_internal_dependency_generator 0
 Summary:        Contains the GNU compiler collection
 Name:           gcc
-Version:        11.2.0
-Release:        7%{?dist}
+Version:        13.2.0
+Release:        1%{?dist}
 License:        GPLv2+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Development/Tools
 URL:            https://gcc.gnu.org/
 Source0:        https://ftp.gnu.org/gnu/gcc/%{name}-%{version}/%{name}-%{version}.tar.xz
-Patch0:         CVE-2023-4039.patch
+#Patch0:         CVE-2023-4039.patch
```

Comment:
Assuming this is not required?  Or tracked for follow-up?

Reply:
This is required for ARM64. I left it as a reminder - the patch needs to be rebased against cc 13.2.0 (which does not have the fix)

Reply:
I've filed a bug to track this: https://microsoft.visualstudio.com/OS/_workitems/edit/47485967